### PR TITLE
Enh readme

### PR DESCRIPTION
**Pull request by MarcCote**
This PR updates the README.md and fixes the turn count (in Inform7).

This PR is based on #124 (I'll rebase once it has been merged).

### CODE REVIEW

File: textworld/generator/inform7/world2inform7.py [hunk 1]
```diff
@@ -768,7 +761,7 @@ def gen_source(self, seed: int = 1234) -> str:
 
         Turning on the extra description command option is an action applying to nothing.
         Carry out turning on the extra description command option:
-            Decrease turn count by 1;
+            Decrease turn count by 1;  [This should not affect turn count.]
```
Comment (tavianator) [inline textworld/generator/inform7/world2inform7.py]: I get this, but on its own this line does look kinda funny ;)

File: README.md [hunk 2]
```diff
@@ -12,12 +12,14 @@ TextWorld requires __Python 3__ and only supports __Linux__ and __macOS__ system
 TextWorld requires some system libraries for its native components.
 On a Debian/Ubuntu-based system, these can be installed with
 
-    sudo apt install build-essential libffi-dev python3-dev curl git
+    sudo apt-get update && sudo apt install build-essential libffi-dev python3-dev curl git
```
Comment (tavianator) [inline README.md]: `apt update` would work

File: README.md [hunk 3]
```diff
@@ -28,89 +30,79 @@ Or, after cloning the repo, go inside the root folder of the project (i.e. along
...
-            Ben Kybartas and
-            Tavian Barnes and
-            Emery Fine and
-            James Moore and
-            Matthew Hausknecht and
-            Layla El Asri and
-            Mahmoud Adada and
-            Wendy Tay and
-            Adam Trischler},
+  author = {Marc-Alexandre C\^ot\'e and \'Akos K\'ad\'ar and Xingdi Yuan and Ben Kybartas and Tavian Barnes and Emery Fine and James Moore and Matthew Hausknecht and Layla El Asri and Mahmoud Adada and Wendy Tay and Adam Trischler},
```
Comment (tavianator) [inline README.md]: This was certainly more readable before, did BibTeX not like it?

Comment (MarcCote) [reply README.md]: I just tried to save on vertical space (I saw some other repo doing that). I'll revert it back.

File: textworld/generator/inform7/world2inform7.py (hunk 1 above)
Comment (MarcCote) [reply textworld/generator/inform7/world2inform7.py]: Haha, I see that now :P. Do you have a suggestion?

Comment (tavianator) [reply textworld/generator/inform7/world2inform7.py]: Maybe "internal framework commands shouldn't count as a turn"?

**Comment by MarcCote** [comment]
Rebased + addressed your comments @tavianator .

**Pull request merged**